src/pkg/packaging/deb/dotnet-sharedframework-debian_config.json

```diff
@@ -30,8 +30,7 @@
 
     "debian_dependencies":{
         "%HOSTFXR_DEBIAN_PACKAGE_NAME%" : {},
-        "libssl1.0.0" : {},
```

Comment:
Confirmed, the resulting package does not exhibit dependency to libssl1.0.0 . 

Depends: libc6 (>= 2.14), libcurl3 (>= 7.16.2), libgcc1 (>= 1:4.1.1), libgssapi-krb5-2 (>= 1.10+dfsg~), liblttng-ust0, libstdc++6 (>= 4.8), libunwind8, libuuid1 (>= 2.16), zlib1g (>= 1:1.1.4), dotnet-hostfxr-2.1.0-preview1-25323-9, libicu52